examples/keyword-extraction-api.ipynb

```diff
@@ -0,0 +1,615 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "be189fde-4e5b-4f80-bae1-ded86a5075a7",
+   "metadata": {
+    "id": "be189fde-4e5b-4f80-bae1-ded86a5075a7"
+   },
+   "source": [
+    "# Keyword extraction with kluster.ai API"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b17a77d9",
+   "metadata": {},
+   "source": [
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/kluster-ai/klusterai-cookbook/blob/main/keyword-extraction-api.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d1d06ea-79c1-4f28-b312-0e5aabe18ff3",
+   "metadata": {
+    "id": "6d1d06ea-79c1-4f28-b312-0e5aabe18ff3"
+   },
+   "source": [
+    "Welcome to the keyword extraction notebook powered by the kluster.ai Adaptive Inference API!\n",
+    "\n",
+    "In this notebook, we’ll demonstrate how to leverage the <a href=\"https://kluster.ai/\" target=\"_blank\">kluster.ai</a> Adaptive Inference API to identify the key terms in a given text. For our demonstration, we use a sample from the AG News dataset to extract the primary keywords from each text. While we’re working with a news dataset, you can easily modify this example to suit your own data and use case. This method allows you to efficiently process datasets of any size, from small samples to large-scale collections, with results categorized by a state-of-the-art language model.\n",
```

Comment:
In this notebook, we’ll demonstrate how to use the kluster.ai API and the Llama 405B Large Language Model (LLM) to identify keywords in a given dataset. By using an extract from the AG News dataset as an example, we’ll show you how to extract keywords from the dataset. You can easily modify this example for your own use case and data format. Our solution efficiently processes text of any size, small text samples to enterprise-scale datasets.